src/Microsoft.AspNet.Mvc.Xml/Formatters/XmlOutputFormatter.cs

```diff
@@ -73,5 +75,27 @@ public static XmlWriterSettings GetDefaultXmlWriterSettings()
         {
             return XmlWriter.Create(writeStream, xmlWriterSettings);
         }
+
+        private Type GetSerializableErrorType(Type type)
+        {
+            if (type == Constants.SerializableErrorType)
+            {
+                return Constants.SerializableErrorWrapperType;
+            }
+
+            return type;
+        }
+
+        internal object GetWrappedSerializableErrorObject(object obj)
```

Comment:
is this marked as internal for testing ? if not I would suggest marking it as protected.